### PR TITLE
chore: use docker for db

### LINKED ISSUE #1: Use docker for Postgres DB, Redis and PgAdmin
Now we have to install a postgres db server to run the dev server.

Use docker image so that we can skip the db installation part and get the dev env working on go with simpler steps.

### PR DESCRIPTION
**Pull request by krushiraj**
- `docker-compose.yml` has config for image and ports, volumes for db
- the data related to db will be stored in `pg-db` in the roor dir of project/
- `docker.sh` - a wrapper script to manage docker containers and related things
- updated credentials in respective files

Closes #1

### CODE REVIEW

**Comment by pbteja1998** [comment]
@krushiraj Since we are setting up docker, can we also add redis to this?

**Comment by krushiraj** [comment]
Yes that is the plan

**Comment by pbteja1998** [comment]
Great. Can you add it to this pr itself or do you think it would need a separate pr?

**Comment by pbteja1998** [comment]
https://github.com/coderplex/learnplex/tree/docs/docs

The files in this needs to be changed.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  postgres:
+    image: 'postgres'
```
Comment (vinaypuppal) [inline docker-compose.yml:4]: Let's use the alpine image which is smaller in size `postgres:10.6-alpine`

**Pull request merged**